README.md

```diff
@@ -113,11 +113,27 @@ pnpm nx build dialtone-documentation
 
 ### Releasing
 
+Running this commands will automatically release all packages that need to be released.
+
+#### Production
+
+Note: needs to be run while on staging branch.
```

Comment:
Slightly more detail about what this command actually does.

Merges staging to production branch
publishes the library package to npm
deploys the documentation site.